js/src/lib/Details/ReadMore.js

```diff
@@ -49,7 +49,7 @@ class ReadMore extends React.Component {
             {collapsed ? text : window.i18n.detail.collapse}
             <img
               src="/assets/detail/ico-readmore.svg"
-              alt=""
+              alt="{window.i18n.detail.readmore_alt}"
```

Comment:
```
alt={window.i18n.detail.readmore_alt}
```

but actually here I think it would be annoying to hear "icon", I specifically added "" to make sure nothing is said